bitnami/pytorch/templates/statefulset.yml

```
@@ -0,0 +1,166 @@
+{{- if eq .Values.mode "distributed" }}
+apiVersion: apps/v1beta2
```

you can use `apiVersion: apps/v1`

```suggestion
apiVersion: apps/v1
```